### PR TITLE
YAY travis...

### PR DESCRIPTION
**Pull request by GitStonic**
it took me days 👍

### CODE REVIEW

File: src/classes/QueryBuilder.js [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
      */
     this._knex = require('knex')({
       client: 'mysql',
-      connection: DB_CREDENTIALS,
+      connection: {
+        user: config.DB_USER,
+        database: config.DB_NAME,
+        password: config.DB_PASS,
+        host: '127.0.0.1'
+      },
```
Comment (alex-taxiera) [inline src/classes/QueryBuilder.js]: Can you just get these vars from `process.env` in this file? there is no need to pass config down here if the environment variables are always available.

File: src/classes/SafeClass.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ class SafeClass {
         if (!expected.has(actual)) errors.push(this._restraintError(key, expected, actual))
       }
     }
-    if (errors.length > 0) throw TypeError('\n\t\t\u0020' + errors.join('\n\t\t\u0020'))
+    if (errors.length !== 0) throw TypeError(errors.join('\n'))
```
Comment (alex-taxiera) [inline src/classes/SafeClass.js]: You got rid of all my cool tabs?

File: src/classes/test/Logger.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+test.afterEach(t => {
+    t.context.log.restore();
+})
+
+test.serial('log', t => {
+    t.context.Logger.log('normal log...')
+    t.true(t.context.log.calledOnce)
+})
+
+test.serial('success', t => {
```
Comment (alex-taxiera) [inline src/classes/test/Logger.js]: Can you please wrap arguments in parentheses? `(t)`

File: src/classes/test/QueryBuilder.js [hunk 4]
```diff
@@ -1,15 +1,9 @@
...
-const DB_CREDENTIALS = {
-  database: 'eris-boiler-test',
-  host: '127.0.0.1',
-  user: 'test',
-  password: 'test123'
-}
 
 test.before(async (t) => {
-  t.context.QueryBuilder = new QueryBuilder(DB_CREDENTIALS, Logger)
+  t.context.QueryBuilder = new QueryBuilder(process.env, Logger)
```
Comment (alex-taxiera) [inline src/classes/test/QueryBuilder.js]: This is super clean.

**Comment by GitStonic** [comment]
i don't like you people

**Pull request merged**